pettingzoo/butterfly/knights_archers_zombies/manual_policy.py

```diff
@@ -0,0 +1,77 @@
+import pygame
+
+
+class ManualPolicy:
+    def __init__(self, env, agent_id: int = 0, show_obs: bool = False):
+
+        self.env = env
+        self.agent_id = agent_id
+        self.agent = self.env.agents[self.agent_id]
+
+        # TO-DO: show current agent observation if this is True
+        self.show_obs = show_obs
+
+        # action mappings for all agents are the same
+        if True:
+            self.default_action = 5
+            self.action_mapping = dict()
+            self.action_mapping[pygame.K_w] = 0  # front
+            self.action_mapping[pygame.K_s] = 1  # back
```

Comment:
Weirdly, moving front and then back does not leave me in the same place. Is this a known feature in the environment?

Reply:
Yes, this is a bug in the KAZ environment currently. A fix for it is in the vector state PR.